configs/pie-components-config/playwright-lit-visual-config.js

```diff
@@ -30,6 +30,11 @@ export function getPlaywrightVisualConfig () {
             /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
             trace: 'on',
             testIdAttribute: 'data-test-id',
+            discovery: {
+                disallowedHostnames: [
```

Comment:
Do you have a link to the docs that explain this?

Reply:
https://www.browserstack.com/docs/percy/common-issue/disallowing-requests

Reply:
Interesting, I'm assuming it still fetches the assets just doesn't rely on the asset discovery process. I wonder if we'll see fewer script failures but more diffs with assets that failed to load.